README.md

```diff
@@ -1,20 +1,24 @@
 # OVAL visualization as graph
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/quality-score.png?b=codeStyle)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=codeStyle) [![Code Coverage](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/coverage.png?b=codeStyle)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=codeStyle) [![Build Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/build.png?b=codeStyle)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/build-status/codeStyle) [![Code Intelligence Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/code-intelligence.svg?b=codeStyle)](https://scrutinizer-ci.com/code-intelligence)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=master) [![Code Coverage](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=master) [![Build Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/build.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/build-status/master) [![Code Intelligence Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/code-intelligence.svg?b=master)](https://scrutinizer-ci.com/code-intelligence)
 
 Understanding result in the blink of an eye
 
 ### How use?
 ```
 Command:
-    python3 run.py arf-file.xml id-rule
+    python3 run.py ARF-file.xml id-rule
 Example:
     python3 run.py data/ssg-fedora-ds-arf.xml xccdf_org.ssgproject.content_rule_disable_host_auth
 ```
-## Requires:
+* *ARF-file.xml* -  ARF xml report from OpenSCAP scan. 
+* *id-rule*  - Rule ID to be visualized. You can input part of ID rule or use regular expression, but if you use in regular expression brackets. You must put regular expression between quotation marks.
```

Comment:
It isn't clear to me what you try to say.

What about this:

Apart from the full rule ID a part of the ID or a regular expression can be used. If brackets are used in the regular expression the regular expression must be quoted.

Reply:
Now it's OK